DOCS/man/mpv.rst

```diff
@@ -328,6 +328,12 @@ g-l
 g-d
     Select an audio device.
 
+g-h
+    Select a file from the watch history. Requires
+    ``--script-opt=select-save_watch_history=yes``. The history path is
```

Comment:
I think this should be native option. The fact that it is implemented in select.lua shouldn't force use to use `--script-opt=select-` which is little bit verbose. Also I consider script-opts to something that should be used to already specific script behavior. Here we have whole separate feature.

Reply:
You mean  only `save_watch_history` should be an option but `watch_history_path` and any future config should be a script-opt?

Reply:
No, I mean all options should be an option.

Reply:
Also on that note, I think the history saving logic should be moved to own script, in case select should be left for selection imho.

Reply:
The saving part is just 17 lines out of 85 though

Reply:
Also I felt that it didn't make sense to separate them when select.lua has to be perfectly aware of which format it was saved in right above